ehitchhiking-ui/src/app/pages/registration/registration.module.ts

```diff
@@ -0,0 +1,16 @@
+import {NgModule} from '@angular/core';
+import {RegistrationComponent} from "./registration.component";
+import {MatButtonModule} from "@angular/material";
+import {AppRoutingModule} from "../../app-routing.module";
+import {FormsModule} from "@angular/forms";
+
+@NgModule({
+  declarations: [RegistrationComponent],
+  imports: [
+    MatButtonModule,
+    AppRoutingModule,
```

Comment:
Are you sure that AppRoutingModule is required here?

Reply:
He shouldn't be here . Should only be in app